Yosemite/Yosemite/Actions/CouponAction.swift

```diff
@@ -67,4 +67,14 @@ public enum CouponAction: Action {
                        pageNumber: Int,
                        pageSize: Int,
                        onCompletion: (Result<Void, Error>) -> Void)
+
+    /// Retrieve a Coupon for a site given the coupon ID
+    ///
+    /// - `siteID`: the site for which coupons should be fetched.
+    /// - `couponID`: ID of the coupon to be retrieve
```

Comment:
super nit:
```suggestion
    /// - `couponID`: ID of the coupon to be retrieved.
```

Reply:
Thank you, fixed in fffda8846e47157b902597dcb3a2b25fc4ce7f96 😅